src/vs/workbench/services/suggest/browser/simpleCompletionItem.ts

```diff
@@ -73,6 +74,7 @@ export class SimpleCompletionItem {
 		// ensure lower-variants (perf)
 		this.labelLow = this.completion.label.toLowerCase();
 		this.labelLowExcludeFileExt = this.labelLow;
+		this.labelLowExcludeTrailingFileSep = this.labelLowExcludeFileExt.replace(/\/$/, '');
 		if (completion.isFile) {
 			if (isWindows) {
 				this.labelLow = this.labelLow.replaceAll('/', '\\');
```

Comment:
This actually won't work properly on Windows. as it could contain `/` or `\` Let's also take `isDirectory` into account for this.

Reply:
For `isWindows` we could normalize the separator here so we only need to check `count(, /)`?

Reply:
I'm wondering if it's weird to even have this set for files, given it'll be the same as `labelLow`.